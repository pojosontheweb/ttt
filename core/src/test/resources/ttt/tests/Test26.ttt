<%!
    // foo bar
    // baz

    /**
     * the calc action bean (used to display result)
     */
    int foo;
%>
comments again...
