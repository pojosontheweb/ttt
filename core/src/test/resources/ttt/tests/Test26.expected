package ttt.tests;

public class Test26 extends com.pojosontheweb.ttt.Template {

	private final int foo;

	/**
	 * Creates an instance of this template.
	 *
	 * @param foo  the calc action bean (used to display result)
	 */
	public Test26(int foo) {
		this.foo = foo;
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "comments again...\n" );
	}
}
